.github/workflows/build.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2022, Kry10 Limited
+# SPDX-License-Identifier: BSD-2-Clause
+
+name: Build
+
+on:
+  pull_request:
+    # Currently aspirational
+    branches:
+      - master
```

Comment:
How aspirational is it? I.e. should we rather leave it out for now?

Reply:
Yeah, should probably leave it out for now. This was just intended as a reminder that we want this on master, but can't easily have that until we've integrated the branches, and the RISC-V branch currently provides a better base for this work.